pkg/ingester/ingester_v2.go

```diff
@@ -240,6 +218,78 @@ func (i *Ingester) v2LabelNames(ctx old_ctx.Context, req *client.LabelNamesReque
 	}, nil
 }
 
+func (i *Ingester) v2MetricsForLabelMatchers(ctx old_ctx.Context, req *client.MetricsForLabelMatchersRequest) (*client.MetricsForLabelMatchersResponse, error) {
+	userID, err := user.ExtractOrgID(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	db := i.getTSDB(userID)
+	if db == nil {
+		return &client.MetricsForLabelMatchersResponse{}, nil
+	}
+
+	// Parse the request
+	from, to, matchersSet, err := client.FromMetricsForLabelMatchersRequest(req)
+	if err != nil {
+		return nil, err
+	}
+
+	// Create a new instance of the TSDB querier
+	q, err := db.Querier(from.Unix()*1000, to.Unix()*1000)
```

Comment:
`nit` prefer `time.Microsecond` here instead

Reply:
It's actually even easier: `from`  and `to` are `model.Time` (not the Go time pkg), and are already expressed in milliseconds, so we've just to cast it to `int64`.

What's your take?

Reply:
that sounds good